addons/escoria-core/game/core-scripts/migrations/esc_migration.gd

```diff
@@ -0,0 +1,27 @@
+# Base class for all migration version scripts. Extending scripts should be
+# named like the version they migrate the savegame to. (e.g. 1.0.0.gd, 1.0.1.gd)
+extends Node
```

Comment:
Why does ESCMigration needs to be a Node? Does it need to be added to a scene, or could it be instanced as an Object instead?

Reply:
Yes, you're right. Probably can go with object. :)

Reply:
Done